components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -117,7 +117,23 @@ function launch_eks() {
 }
 
 function delete_eks() {
-  eksctl delete cluster --name "${EKS_CLUSTER_NAME}" --region "${REGION}"
+  time_unit=m
+  timeout=60
+  retry_interval=5
+
+  while [ "$timeout" -gt "0" ]; do
+    eksctl delete cluster --name "$EKS_CLUSTER_NAME" --region "$REGION" --wait
+    if [ $? -eq 0 ]; then
+      echo "EKS cluster deleted"
+      return 0
+    fi
+    echo "Failed to delete EKS cluster. Retrying after ${retry_interval}${time_unit}"
+    timeout=$(($timeout-$retry_interval))
```

Comment:
whats changing between these retries, as in, is the stack left in a state after first command where retry could help?

why retry "12" times, its a big number for retrying in this case

Reply:
The "i/o timeout" error did not occur when i tried to manually cleanup the cluster afterwards.
(So i assume cluster is unstable right after Minio failure and we need to wait for the EKS cluster to recover)
Need to find the root cause.

I just wanted it to retry for an hour.
We can change it later once we have stats about how long it takes on average.